src/renderer/components/layout/sidebar.tsx

```diff
@@ -52,8 +51,6 @@ import { SidebarItem } from "./sidebar-item";
 
 interface Props {
   className?: string;
-  compact?: boolean; // compact-mode view: show only icons and expand on :hover
-  toggle(): void; // compact-mode updater
```

Comment:
Why remove this? It is a nice feature to have, I think it would be great to incorporate the icons into the catalog's sidebar too since each Catalog category will have an icon.

Reply:
It might seem like nice feature but it was always produced confusion from developer side along with user one. It's hard to maintain persistent and meaningful navigation and icon position for different modes (expanded/collapsed). See this for example https://github.com/lensapp/lens/issues/2873#issuecomment-849793825